models/cut_semantic_mask_model.py

```diff
@@ -34,6 +34,8 @@ def modify_commandline_options(parser, is_train=True):
         parser.add_argument('--contrastive_noise', type=float, default=0.0, help='noise on constrastive classifier')
         parser.add_argument('--lambda_sem', type=float, default=1.0, help='weight for semantic loss')
 
+        parser.add_argument('--second_f_s',action='store_true', help='whether to use a light (unet) network for f_s')
```

Comment:
`--disjoint_f_s` ? and it may not be a `light` nn, right ? We may want the two f_s to have the exact same architecture, via a single `--net_f_s` option ?